projects/step-frontend/src/lib/modules/_common/components/sidebar/sidebar.component.ts

```diff
@@ -99,21 +99,22 @@ export class SidebarComponent implements AfterViewInit, OnDestroy {
         this._authService.gotoDefaultPage();
         break;
       default:
-        if (this._ajsLocation.path().includes('/root/' + viewId)) {
```

Comment:
not sure what has this been used for, but I didn't really see an issue removing it

Reply:
We still require this. It reloads the current page, i.e. when you are currently on plans/list and click plans in the menu without this it will not reload "plans/list" since no URL change is triggered.